### ISSUE TITLE
duda con los test cases privados T3

### ISSUE THREAD

**Issue by TomasAguirrebena** [IIC2233/Syllabus#609]
### Prerrequisitos
(Marcar colocando una `X` entre los corchetes los ítems que ya hiciste, así: `"[X]"`)

* [x ] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/3.-Reglas-del-foro-(issues)).
* [x] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues).
* [x ] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al ayudante de Bienestar o al correo del curso (coordinacion.iic2233@gmail.com).
* [x ] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.

### Contenido

tengo la duda de si es normal que no pueda ejecutar los test cases desde dentro de la misma carpeta de los test ya que me da el error 
ModuleNotFoundError: No module named 'consultas' asumire que es que de algun mododo esta buscando mal, y probe mas metodos y agregre a todos_tests uno a uno los test cases para poder ejecutarlos de la manera:

from correccion_privados.tests_privados.test_00_animales_segun_edad_correctitud import TestAnimalesSegunEdadCorrectitud

ahora puedo ejecutarlos pero al momento de ejecutar los que no son de correctitud osea de ejecución me da el error:
 
![image](https://github.com/IIC2233/Syllabus/assets/127432854/eab69a06-bd43-4762-a8eb-db82a3f4f565)

vi que alguien en la issue anterior a mi tenia problemas con los de tiempo de ejecución pero no se si era exactamente lo mismo, por lo que opte por abrir una issue nueva

**Comment by mpiavf** [comment]
Hola!
Te recomiendo que copies la carpeta de tests privados a la misma altura que los tests públicos. Además asegurate de reemplazar "data" y "tests_solution.py" con la nueva versión que está en el Syllabus. Finalmente posiciona tu terminal dentro de la carpeta "tests_privados" y vas ejecutando el tests en particular que a ti te interesa. Similar a cómo lo hacemos con las actividades.
Si no te funciona de esta forma me comentas !!

**Comment by TomasAguirrebena** [comment]
hola gracias! ya pude corregir mi tarea la tabla de puntajes supongo es la misma que teníamos antes, y tengo una duda cuando llega el comentario de canvas con los datos que nos bajan la nota y etc ? por que los test que tengo no cuadran para nada con la nota que tengo y el test de elementos prohibidos no da ningún false entonces no se que habrá pasado.

**Comment by mpiavf** [comment]
Sí, pude darme cuenta de que el commit a revisar y recopilado no coincide con el último que tienes, por favor manda un formulario de recorrección en donde expliques lo sucedido, para que se tenga en cuenta tu caso y se vea que ocurrió.

**Comment by TomasAguirrebena** [comment]
en mi canvas si no me equivoco si aparece correctamente el commit es internamente que esta mal ? (se que no deberia hacer muchas preguntas de casos particulares pero es que quiero aligerarme las dudas)

**Comment by mpiavf** [comment]
Hola @TomasAguirrebena  (y a cualquiera otra persona que le pase este problema).

Si en Canvas te aparece el commit correcto, con dicho commit probaste tu tarea y la cantidad de tests privados que apruebas no se condice para nada con el feedback entregado, es que efectivamente **pudo haber sido** un error interno de nuestro lado. Por lo tanto, por favor debes mandar a recorregir para identificar y volver a ejecutar los casos que presentan dicho error. De este modo poder actualizar su nota, o bien explicarles con más detalle por qué la nota estaría correcta. 

Por si acaso, digo que "pudo haber sido" porque hay casos de tareas que no funcionaron en otro computador porque usaron paths absolutos, paths exclusivos de windows, hacían import de una librería prohibida y que el computador no tiene, hacían import de un archivo que no subieron a su repo, etc. Por lo tanto, pueden existir casos donde el estudiante pasa los tests, pero eso es únicamente posible en su PC y en ningún otro, pero igual instamos que manden a recorregir para poder explicarles mejor el error, o bien actualizar la nota si fue un problema interno nuestro.
